helpers/helper-translation-discussion.php

```diff
@@ -214,6 +214,23 @@ public function sanitize_comment_topic( $comment_topic ) {
 		return $comment_topic;
 
 	}
+
+	public function sanitize_comment_locale( $comment_locale ) {
+		$gp_locales     = new GP_Locales();
+		$all_gp_locales = array_keys( $gp_locales->locales );
+
+		if ( ! in_array( $comment_locale, $all_gp_locales ) ) {
+			$comment_locale = 'unknown';
```

Comment:
Couldn't we just fall back to `false`, too? Then the extra check for `unknown` below won't be necessary.

Reply:
But this might make it difficult to identify comments whose locales were forged/invalid because this suggested change would assume that the comments were made for the English discussion page as `add_comment_meta()` stores boolean `false` as an empty string.

Reply:
Ok, let's go for an empty string. So:
```suggestion
			$comment_locale = '';
```